<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cloudList</class>
 <widget class="QWidget" name="cloudList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="playAll">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#playAll{
	background-color : rgba(3,122,250,0.8);
	color : white;
	border-radius:10px;
	border : 0px;
	padding-left : 10px;
	padding-right : 10px;
	padding-top:4px;
	padding-bottom : 4px;
}
#playAll:hover{
	background-color : rgba(3,122,250,0.9);
}
#playAll:pressed{
	background-color : rgba(3,122,250,1);
}</string>
   </property>
   <property name="text">
    <string>播放全部</string>
   </property>
   <property name="icon">
    <iconset resource="resourece.qrc">
     <normaloff>:/pic/playSelected.png</normaloff>:/pic/playSelected.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchText">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#searchText{
	background-color:#E0E0E0;
	border : 0px;
	border-radius:15px;
	padding-left : 10px;
	padding-right : 30px;
}</string>
   </property>
   <property name="placeholderText">
    <string>搜索歌单音乐</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>云上音乐</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="musicList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>760</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#musicList{
	border : 0px;
	background-color:white;
	border-radius:10px;
	color : #333333;
}

#musicList::item::selected{
    background:#E6E6E6;
}

#musicList::item:hover{
    background:#F1F2F3;
}

</string>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>音乐标题</string>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>51</red>
       <green>51</green>
       <blue>51</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>歌手</string>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>51</red>
       <green>51</green>
       <blue>51</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>专辑</string>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>51</red>
       <green>51</green>
       <blue>51</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>时长</string>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>51</red>
       <green>51</green>
       <blue>51</blue>
      </color>
     </brush>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="searchIcon">
   <property name="geometry">
    <rect>
     <x>755</x>
     <y>63</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#searchIcon{
	background-color : rgba(0,0,0,0);
	border : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resourece.qrc">
     <normaloff>:/pic/search.png</normaloff>:/pic/search.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resourece.qrc"/>
 </resources>
 <connections/>
</ui>
